### PR TITLE
Feature/backporting

### PR DESCRIPTION
**Pull request by florianow**
maybe we could make the first release out of it?

### CODE REVIEW

File: kit/azure/bootstrap/resources.docs-spn.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# todo: this could be nicer packaged as it's own submodule, so that it doesn't need all those count = ... checks
+
+resource "azurerm_user_assigned_identity" "docs" {
```
Comment (JohannesRudolph) [inline kit/azure/bootstrap/resources.docs-spn.tf:3]: f: I'm afraid we can't or should not backport this 1:1 from livkid bank as it has some hardcoded assumptions in there that need proper modularization.

File: kit/azure/landingzones/corp-online/lib/online/policy_definitions/placeholder.json.template
Comment (JohannesRudolph) [inline kit/azure/landingzones/corp-online/lib/online/policy_definitions/placeholder.json.template:1]: what's the purpose of this one? i assume it's been there all along

File: kit/azure/landingzones/sandbox/template/platform-module/terragrunt.hcl [hunk 2]
```diff
@@ -26,7 +26,7 @@ provider "azurerm" {
   subscription_id = "${include.platform.locals.platform.azure.subscriptionId}"
   client_id       = "${dependency.bootstrap.outputs.client_id}"
   client_secret   = "${dependency.bootstrap.outputs.client_secret}"
-}
+  }
```
Comment (JohannesRudolph) [inline kit/azure/landingzones/sandbox/template/platform-module/terragrunt.hcl:29]: s: the previous formatting was besser, revert this line :-)

File: kit/azure/billing/template/platform-module/terragrunt.hcl [hunk 3]
```diff
@@ -36,13 +36,11 @@ provider "azuread" {
...
 }
 
-
 inputs = {
   # todo: set input variables
   scope = "${dependency.organization-hierarchy.outputs.parent_id}"
   budget_time_period = [{
-    start = "2022-06-01T00:00:00Z",
-    end   = "2022-07-01T00:00:00Z"
+    start = "2023-11-01T00:00:00Z"
```
Comment (JohannesRudolph) [inline kit/azure/billing/template/platform-module/terragrunt.hcl:43]: d: iirc azure will throw an error if date is in a past month, so we should add a #todo here on that line to adjust it

File: kit/azure/bootstrap/README.md
Comment (JohannesRudolph) [inline kit/azure/bootstrap/README.md:1]: revert, docs UAMI is not ready for inclusion in collie hub yet

File: kit/azure/bootstrap/resources.docs-spn.tf
Comment (JohannesRudolph) [inline kit/azure/bootstrap/resources.docs-spn.tf:1]: I think we can just revert the whole file,  docs UAMI is not ready for inclusion in collie hub yet

File: kit/azure/bootstrap/variables.tf [hunk 4]
```diff
@@ -4,6 +4,12 @@ variable "service_principal_name" {
   default     = "cloudfoundation_tf_deploy_user"
 }
 
+variable "documentation_uami_name" {
```
Comment (JohannesRudolph) [inline kit/azure/bootstrap/variables.tf:7]: f: revert, docs UAMI is not ready for inclusion in collie hub yet

**Review by JohannesRudolph — APPROVED** [review]
biggest change I suggest is keeping the GH Actions docs generation pipeline out of collie hub, this needs some more work

File: kit/azure/bootstrap/resources.docs-spn.tf (hunk 1 above)
Comment (florianow) [reply kit/azure/bootstrap/resources.docs-spn.tf:3]: I took it complete out of the hub yet

File: kit/azure/billing/template/platform-module/terragrunt.hcl (hunk 3 above)
Comment (florianow) [reply kit/azure/billing/template/platform-module/terragrunt.hcl:43]: true it throws an error. I add an comment.

**Pull request merged**